gobblin-runtime/src/main/java/org/apache/gobblin/scheduler/JobScheduler.java

```diff
@@ -612,7 +613,10 @@ public void executeImpl(JobExecutionContext context)
       long triggerTimestampMillis = trigger.getPreviousFireTime().getTime();
       jobProps.setProperty(ConfigurationKeys.SCHEDULER_EVENT_TO_TRIGGER_TIMESTAMP_MILLIS_KEY,
           String.valueOf(triggerTimestampMillis));
-
+      LOG.info("Scheduler trigger validation: [flowName: {} flowGroup: {}] - triggerTime: {} nextTriggerTime: {} - "
+              + "Job triggered by scheduler",
```

Comment:
the other one said "job scheduled" and this one says "job triggered by scheduler".

is this one not also being scheduled?  if it is, suggest:
"job newly scheduled"
"job re-scheduled"

Reply:
Same comment here, the job name and job group don't map to flownames and flowgroups exactly.
If you want to make it GaaS Specific, it has to live in the `GobblinServiceJobScheduler`

Reply:
I'll update the first comment to say newly scheduled, but this second instance is not re-scheduled but rather the callback invoked when the Scheduler trigger fires and the job is actually run. This is the important part we want to validate. However, I may need to override the super class method by the `GobblinServiceJobScheduler` to retrieve the flow properties.